time/slots/slottime.go

```diff
@@ -265,7 +265,7 @@ func SyncCommitteePeriodStartEpoch(e primitives.Epoch) (primitives.Epoch, error)
 // given slot start time
 func SecondsSinceSlotStart(s primitives.Slot, genesisTime, timeStamp uint64) (uint64, error) {
 	if timeStamp < genesisTime+uint64(s)*params.BeaconConfig().SecondsPerSlot {
-		return 0, errors.New("could not compute seconds since slot start: invalid timestamp")
+		return 0, fmt.Errorf("could not compute seconds since slot %d start: invalid timestamp, got %d < want %d", s, timeStamp, genesisTime+uint64(s)*params.BeaconConfig().SecondsPerSlot)
```

Comment:
this error still happens Occasionally on e2e startup
```
 helpers.go:242: time="2024-11-05 21:18:14" level=error msg="could not check if block arrived early" error="could not compute seconds since slot 0 start: invalid timestamp, got 1730841480 < want 1730841484" prefix=forkchoice-doublylinkedtree
    helpers.go:242: time="2024-11-05 21:18:14" level=error msg="Could not get builder payload" error="builder returned header with 0 bid amount" prefix="rpc/validator"
    ```